documentation/rosetta-modelling-component.md

```diff
@@ -1530,23 +1540,25 @@ Mappings are expected to be one-to-one with each input value mapping to one Rose
 
 The `meta` keyword inside a synonym is used to map [metadata](#meta-data-annotation).
 
-In the below example, the value of the "issuer" input will be mapped to the value of the `issuer` attribute and the value of the "issuerIdScheme" metadata associated to "issuer" will be mapped to the `scheme` metadata attribute.
+In the below example, the value of the "combustible" input will be mapped to the value of the `fuelType` attribute and the value of the "scheme" metadata associated to "combustible" will be mapped to the `scheme` metadata attribute.
 
 ```
-issuer string (0..1)
-  [metadata scheme]
-  [synonym FpML_5_10 value "issuer" meta "issuerIdScheme"]
+type EngineSpecification:
+    fuelType string (1..*)
+        [metadata scheme]
+        [synonym META_SCHEME_EXAMPLE_1 value "combustible" meta "scheme"]
 ```
 
 #### Enumeration
 
-A synonym on an enumeration provides mappings from the string values in the input document to the values of the enumeration. E.g. the FpML value `Broker` will be mapped to the enumeration value `NaturalPersonRoleEnum.Broker` in Rosetta:
+A synonym on an enumeration provides mappings from the string values in the input document to the values of the enumeration. E.g. the FpML value `HYBRID` will be mapped to the enumeration value `EngineEnum.HYBRID` in Rosetta:
 
 ```
-enum NaturalPersonRoleEnum: <"The enumerated values for the natural person's role.">
-
-  Broker <"The person who arranged with a client to execute the trade.">
-    [synonym FpML_5_10 value "Broker"]
+enum EngineEnum: <"The enumerated values for the natural person's role.">
```

Comment:
Unless you have good rationale to use HYBRID? Please elaborate as I have made many similar comments below.

Reply:
no, that was because in the 1st document of the mapping taxonomy you send when I was doing the taxonomy change (the word doc at the beggining of the project) in the samples in some cases you specified HYBRID in capital letters and I let it te same in the tests. Will change that for this documentation and the samples afected in translate-demo